### ISSUE TITLE
Invalid owner ID exception due to usage tab

### ISSUE THREAD

**Issue by Mosnar** [verbb/formie#1157]
### Describe the bug

I'm receiving an "Invalid owner ID" error when trying to edit a form.

```
yii\base\InvalidConfigException: Invalid owner ID: 192390 in /var/www/html/cms/vendor/craftcms/cms/src/elements/MatrixBlock.php:328
Stack trace:
#0 /var/www/html/cms/vendor/twig/twig/src/Extension/CoreExtension.php(1607): craft\elements\MatrixBlock->getOwner()
#1 /var/www/html/cms/vendor/craftcms/cms/src/helpers/Template.php(101): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(craft\elements\MatrixBlock), 'getOwner', Array, 'method', false, false)
#2 /var/www/html/cms/storage/runtime/compiled_templates/ef/ef39c70e3feeb890cbd840f2e5e09a32.php(168): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(craft\elements\MatrixBlock), 'getOwner', Array, 'method')
#3 /var/www/html/cms/vendor/twig/twig/src/Extension/CoreExtension.php(1202): __TwigTemplate_bdf32de1b8eee6ba05c3fa71e5e4147b->macro_tableColumns(Object(craft\elements\MatrixBlock), Object(verbb\formie\fields\Forms), Array)
#4 /var/www/html/cms/storage/runtime/compiled_templates/ef/ef39c70e3feeb890cbd840f2e5e09a32.php(94): twig_call_macro(Object(__TwigTemplate_bdf32de1b8eee6ba05c3fa71e5e4147b), 'macro_tableColu...', Array, 69, Array, Object(Twig\Source))
#5 /var/www/html/cms/vendor/twig/twig/src/Template.php(394): __TwigTemplate_bdf32de1b8eee6ba05c3fa71e5e4147b->doDisplay(Array, Array)
#6 /var/www/html/cms/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#7 /var/www/html/cms/vendor/twig/twig/src/TemplateWrapper.php(47): Twig\Template->display(Array, Array)
#8 /var/www/html/cms/storage/runtime/compiled_templates/d5/d5360b8613fceb49564568fef60202f7.php(263): Twig\TemplateWrapper->display(Array)
#9 /var/www/html/cms/vendor/twig/twig/src/Template.php(171): __TwigTemplate_070f80bde5567b04cc127043e284eb99->block_content(Array, Array)
#10 /var/www/html/cms/storage/runtime/compiled_templates/10/10cdc91ecf15107625c648a190e19556.php(590): Twig\Template->displayBlock('content', Array, Array)
#11 /var/www/html/cms/vendor/twig/twig/src/Template.php(171): __TwigTemplate_8ff86f59a38314483246723c1c22f2a5->block_main(Array, Array)
#12 /var/www/html/cms/storage/runtime/compiled_templates/10/10cdc91ecf15107625c648a190e19556.php(374): Twig\Template->displayBlock('main', Array, Array)
#13 /var/www/html/cms/vendor/twig/twig/src/Template.php(171): __TwigTemplate_8ff86f59a38314483246723c1c22f2a5->block_body(Array, Array)
#14 /var/www/html/cms/storage/runtime/compiled_templates/61/611fa0fcbdf9dc8d9c567b2bf925570a.php(94): Twig\Template->displayBlock('body', Array, Array)
#15 /var/www/html/cms/vendor/twig/twig/src/Template.php(394): __TwigTemplate_aea17e9063b38b670db503e0dcce5347->doDisplay(Array, Array)
#16 /var/www/html/cms/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#17 /var/www/html/cms/storage/runtime/compiled_templates/55/55c5d015d6c4f74c05da738ca8569bdd.php(56): Twig\Template->display(Array, Array)
#18 /var/www/html/cms/vendor/twig/twig/src/Template.php(394): __TwigTemplate_4d36d0294ff971bd422cc5ee35d5ef47->doDisplay(Array, Array)
#19 /var/www/html/cms/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#20 /var/www/html/cms/storage/runtime/compiled_templates/10/10cdc91ecf15107625c648a190e19556.php(179): Twig\Template->display(Array, Array)
#21 /var/www/html/cms/vendor/twig/twig/src/Template.php(394): __TwigTemplate_8ff86f59a38314483246723c1c22f2a5->doDisplay(Array, Array)
#22 /var/www/html/cms/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#23 /var/www/html/cms/storage/runtime/compiled_templates/d5/d5360b8613fceb49564568fef60202f7.php(59): Twig\Template->display(Array, Array)
#24 /var/www/html/cms/vendor/twig/twig/src/Template.php(394): __TwigTemplate_070f80bde5567b04cc127043e284eb99->doDisplay(Array, Array)
#25 /var/www/html/cms/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#26 /var/www/html/cms/vendor/twig/twig/src/Template.php(379): Twig\Template->display(Array)
#27 /var/www/html/cms/vendor/twig/twig/src/TemplateWrapper.php(40): Twig\Template->render(Array, Array)
#28 /var/www/html/cms/vendor/twig/twig/src/Environment.php(277): Twig\TemplateWrapper->render(Array)
#29 /var/www/html/cms/vendor/craftcms/cms/src/web/View.php(428): Twig\Environment->render('formie/forms/_e...', Array)
#30 /var/www/html/cms/vendor/craftcms/cms/src/web/View.php(481): craft\web\View->renderTemplate('formie/forms/_e...', Array)
#31 /var/www/html/cms/vendor/craftcms/cms/src/web/TemplateResponseFormatter.php(56): craft\web\View->renderPageTemplate('formie/forms/_e...', Array, 'cp')
#32 /var/www/html/cms/vendor/yiisoft/yii2/web/Response.php(1095): craft\web\TemplateResponseFormatter->format(Object(craft\web\Response))
#33 /var/www/html/cms/vendor/craftcms/cms/src/web/Response.php(286): yii\web\Response->prepare()
#34 /var/www/html/cms/vendor/yiisoft/yii2/web/Response.php(339): craft\web\Response->prepare()
#35 /var/www/html/cms/vendor/yiisoft/yii2/base/Application.php(390): yii\web\Response->send()
#36 /var/www/html/cms/web/index.php(12): yii\base\Application->run()
#37 {main}
```

### Steps to reproduce

1. Create an entry with a matrix block linked to a Formie form.
2. Delete the entry
3. Attempt to access the form

### Form settings

Not sure of form settings, as I can't access the form.

### Craft CMS version

Craft Pro 4.2.8

### Plugin version

4.0.14

### Multi-site?

No

**Comment by Mosnar** [comment]
Quick update - I was able to get the form working again by running garbage collection.

**Comment by Mosnar** [comment]
The problem occurs when a soft-deleted entry contains a matrix block with a Formie form.

**Comment by engram-design** [comment]
Someone did report this earlier, but I was unable to replicate. I still can't seem to replicate that with a soft-deleted entry that contains a Matrix block, referencing a Formie form.

**Comment by engram-design** [comment]
Aha, finally able to replicate it now - somehow! Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.14",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.15](https://github.com/verbb/formie/releases/tag/2.0.15)
